roles/rke2/tasks/nodeport_proxy.yaml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Copy http proxy systemd service file
+  ansible.builtin.template:
+    src: templates/nodeport/proxy/http-proxy.service.j2
+    dest: /etc/systemd/system/http-proxy.service
+    mode: '0644'
+
+- name: Copy https proxy systemd service file
+  ansible.builtin.template:
+    src: templates/nodeport/proxy/https-proxy.service.j2
+    dest: /etc/systemd/system/https-proxy.service
+    mode: '0644'
+
+- name: Reload systemd daemon
+  ansible.builtin.systemd:
+    daemon_reload: yes
```

Comment:
This should be a handler that is notified from
- Copy http proxy systemd service file
- Copy https proxy systemd service file

You can then use

```
- meta: flush_handlers
```

To cause the handler to reload immediately (so that it can be used by the following service enable/start).

This makes it so that systemd isn't reloaded every time the role is executed: only if the unit files change.